packages/fxa-content-server/app/scripts/templates/post_verify/finish_account_setup/set_password.mustache

```diff
@@ -1,5 +1,5 @@
 <div id="main-content" class="card account-setup-set-password">
-    {{#isLinkValid}}
+    {{^isLinkValid}}
```

Comment:
Was this swapped for clarity?

Reply:
Yes. `isLinkValid` was actually false here when the link was valid and that was just confusing.

So I changed:
`isLinkValid: !this._verificationInfo.isValid()` --> `isLinkValid: this._verificationInfo.isValid()`
and updated the mustache.